### ISSUE TITLE
Either Short URL or .me URL isn't picked up to warn on changes

### ISSUE THREAD

**Issue by garretthyder** [GlotPress/GlotPress-WP#728]
Hello,

I get the warning when I modify a URL to localize it all of the time. Was wondering how it determined there was a URL in the string? Mainly as I think it's either ignoring short urls or .me URLs.

You can test with this one;
https://translate.wordpress.org/projects/wp-plugins/jetpack/dev-readme/en-ca/default?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=3061269&filters%5Btranslation_id%5D=49918867

Basically if you switch it to https you should get the warning, but I don't so I'm wondering if it's because it's a .me URL or if it's because it's a short url that it's not picked up.

Either way they'll want to accommodate URLS like - http://wp.me/p1moTy-5hK

All the best,
Cheers

**Comment by ocean90** [comment]
Warnings for URLs are a provided by a custom plugin on translate.w.org. You'll find the related code here: https://meta.trac.wordpress.org/browser/sites/trunk/wordpress.org/public_html/wp-content/plugins/wporg-gp-custom-warnings/wporg-gp-custom-warnings.php?rev=3390&marks=51-60#L27

tldr; translate.w.org ignores https <=> http changes.

**Comment by garretthyder** [comment]
Thanks @ocean90 that was definitely it (https vs http) so will drop as it's expected behaviour.
And sorry I logged in the wrong place again
